ethcore/sync/src/blocks.rs

```diff
@@ -196,11 +196,11 @@ impl BlockCollection {
 	}
 
 	/// Insert a collection of block receipts for previously downloaded headers.
-	pub fn insert_receipts(&mut self, receipts: Vec<Bytes>) -> Vec<Vec<H256>> {
+	pub fn insert_receipts(&mut self, receipts: &[&[u8]]) -> Vec<Vec<H256>> {
```

Comment:
hmm, I didn't understand this first.

Personally, I would prefer to have `Vec<Bytes>` here and just move each receipt in `fn insert_receipt` and don't clone it there if that's possible (seems so) but I don't have any strong opinions on it so do what you want.